### PR TITLE
[eZ Launchpad] Add installation on the homepage

### PR DESCRIPTION
**Pull request by Plopix**
Here is the PR for the homage and the eZ Launchpad installation process.

Code is ready for you to add a new FieldDefinition `launchpad_description` to manage specific text from the admin.

screen shot 2017-11-01 at 11 34 39 pm
screen shot 2017-11-01 at 11 42 25 pm

Logo for Platform.sh changed as well.
screen shot 2017-11-01 at 11 34 48 pm

### CODE REVIEW

File: web/assets/themes/ezplatform/images/ez-launchpad.svg [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SylvainGuittard) [inline web/assets/themes/ezplatform/images/ez-launchpad.svg]: Could you please replace the current code by this?
```

Open SVG

Paste markup
Demo
Contribute
About
Powered by the incredible SVGO


IMAGE
CODE
2.12k - 25.2% saving
<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 84 79.9"><style>.st0{opacity:0.15;fill:#56352C;} .st1{fill:#FCB812;} .st2{fill:#514744;} .st3{fill:#FFDEAC;} .st4{fill:#F4EDEA;} .st5{fill:#F15A22;} .st6{fill:#E2491A;} .st7{fill:#FFFFFF;} .st8{fill:#15B4CE;} .st9{fill:#009BB1;} .st10{fill:#DA5221;} .st11{opacity:0.1;}</style><path class="st0" d="M78.9 7.8c.1-.4.1-.8.1-1.1v-.2-1-.3-1-.3V3L25.7 56.1l5.8 5.8 2.8-2.8.6.6c9.2-3.1 17.1-7.3 25.8-16C72.6 31.8 77.9 19.6 78.9 7.8z"/><path class="st1" d="M31.1 59.2c-1.1 2.9-4.7 10.7-11.5 14.9-7 4.4-15.2 4.5-15.5 3.7-.2-.5 3.4-.8 5-3.7 1.5-2.9-.3-5.4.9-10.5.1-.3.6-2.4 1.7-4.5 2.6-4.7 7.6-7.7 10.3-8.9l9.1 9z"/><path class="st2" d="M3.5 78.5c-.2-.2-.2-.3-.3-.5-.3-1 .7-1.4 1.4-1.6 1.1-.4 2.8-1.1 3.7-2.8.7-1.3.6-2.4.5-4.1-.1-1.6-.3-3.6.3-6.2.1-.4.6-2.6 1.8-4.8 3-5.6 9-8.5 10.8-9.3l.6-.2 9.9 9.9-.2.6c-.7 1.7-4.3 10.6-11.9 15.3-6.6 4.1-14.2 4.7-16 4-.3-.1-.4-.2-.6-.3zm18.4-27.3c-3.3 1.5-7.3 4.4-9.3 8.2-1.1 2-1.6 3.9-1.6 4.3-.6 2.4-.4 4.1-.3 5.7.1 1.7.2 3.3-.7 5.1-.7 1.3-1.7 2.2-2.7 2.7 2.8-.2 7.5-1.2 11.9-3.9 6.5-4 9.9-11.4 10.9-13.9l-8.2-8.2z"/><path class="st3" d="M28.9 57.5c-4.7 9.8-9.3 11.5-11.2 11.9-1.2.3-3.5.5-3.7.1-.1-.4 1.5-.8 2.2-2.2.6-1.4-.1-2.9-.3-3.6-.5-1.8.5-5.1 7.7-11.2l5.3 5z"/><path class="st4" d="M22.482 47.278l11.852 11.906-2.764 2.752L19.718 50.03z"/><path class="st2" d="M30.8 62.6L19 50.7c-.4-.4-.4-.9 0-1.3l2.8-2.8c.2-.2.4-.3.6-.3.2 0 .5.1.6.3l11.8 11.9c.2.2.3.4.3.6 0 .2-.1.5-.3.6L32 62.5c-.3.4-.8.4-1.2.1zm-9.9-12.5l10.5 10.6 1.5-1.5-10.5-10.6-1.5 1.5z"/><path class="st5" d="M31.7 49.8c7.1 7.1 9.2 17.4 6.3 26.4-.3.9.7 1.7 1.5 1.3 2-1.1 3.8-2.5 5.5-4.2 10.6-10.6 10.2-28-1.3-38-9.5-8.3-23.9-8.5-33.6-.5-2.5 2.1-4.5 4.5-6 7.1-.5.8.3 1.8 1.2 1.5 9.1-2.9 19.3-.7 26.4 6.4z"/><path class="st6" d="M43.8 35.2c-3.6-3.1-7.8-5.1-12.2-5.9 3.7 1 7.2 2.8 10.2 5.5 11.5 10 11.9 27.5 1.3 38-1.5 1.5-3.2 2.8-5 3.9.1.7.9 1.2 1.5.8 2-1.1 3.8-2.5 5.5-4.2 10.6-10.6 10.1-28-1.3-38.1z"/><path class="st7" d="M5.3 43c1.5-2.6 3.5-5.1 6-7.1 9.7-8.1 24.1-7.9 33.6.5.3.2.5.5.8.7-.6-.6-1.2-1.3-1.9-1.9-9.5-8.3-23.9-8.5-33.6-.5-2.5 2.1-4.5 4.5-6 7.1-.4.7.2 1.6 1 1.5-.1 0 0-.1.1-.3z"/><path class="st2" d="M37.6 77.9c-.5-.5-.7-1.3-.5-2 2.9-9 .6-18.8-6.1-25.5-6.7-6.7-16.4-9-25.4-6.2-.8.2-1.6 0-2.1-.6s-.5-1.4-.1-2.1c1.6-2.8 3.7-5.3 6.2-7.4 10-8.3 25-8.1 34.8.5 5.7 5 9 11.9 9.2 19.4.2 7.4-2.6 14.7-7.9 19.9-1.7 1.7-3.6 3.2-5.7 4.3-.7.4-1.6.3-2.2-.2-.1 0-.1 0-.2-.1zm-5.3-28.7s.1 0 0 0c7.2 7.2 9.7 17.7 6.6 27.3 0 .1 0 .1.1.2s.1 0 .1 0c1.9-1.1 3.7-2.5 5.3-4 4.9-5 7.6-11.7 7.4-18.7-.2-7-3.3-13.5-8.6-18.1-9.1-8-23.1-8.2-32.5-.5-2.4 2-4.3 4.3-5.8 6.9v.1c0 .1.1.1.2.1 9.7-3 20.1-.5 27.2 6.7z"/><path class="st4" d="M21.2 47.4c-.2-.2-.3-.6-.2-.9 3.6-10.6 8.3-18.2 16.3-26.1C50.3 7.4 65.1.8 79.1 1.9c.2 0 .4.1.6.3.1.1.2.4.3.6 1 13.9-5.7 28.8-18.7 41.7-8 7.9-15.5 12.6-26.1 16.2-.3.1-.7 0-.9-.2L21.2 47.4z"/><path class="st2" d="M79 2.8c.9 12.6-4.6 27.3-18.4 41-8.7 8.6-16.6 12.9-25.8 16-2-2-10.9-11-12.9-13 3.1-9.3 7.3-17.2 16-25.8C51.7 7.3 66.4 1.8 79 2.8m1.3-1.3c-.3-.3-.7-.5-1.2-.5-14.2-1-29.3 5.6-42.5 18.7-8.1 8-12.8 15.7-16.5 26.4-.2.7-.1 1.4.4 1.9l12.9 13c.5.5 1.2.7 1.9.4C46.1 57.8 53.7 53 61.8 45c13.3-13.1 20-28.1 19-42.4 0-.4-.2-.8-.5-1.1z"/><path class="st5" d="M65.9 15.8c2.8 2.9 5.9 6.1 8.5 9.1 3.7-7.7 5.1-15.6 4.6-22.1-6.5-.5-14.2.8-21.9 4.4 2.9 2.5 6 5.7 8.8 8.6z"/><path class="st8" d="M50.4 31.3c-2.9-2.9-2.9-7.7 0-10.6 2.9-2.9 7.6-2.9 10.6 0 2.9 2.9 2.9 7.7 0 10.6-3 2.9-7.7 2.9-10.6 0z"/><path class="st2" d="M60.3 21.4c2.6 2.6 2.6 6.7 0 9.3-2.6 2.6-6.7 2.5-9.3 0-2.6-2.6-2.5-6.7 0-9.3 2.6-2.6 6.7-2.6 9.3 0m1.3-1.3c-3.3-3.3-8.6-3.3-11.8 0-3.3 3.3-3.3 8.6 0 11.8 3.3 3.3 8.6 3.3 11.8 0 3.2-3.2 3.3-8.5 0-11.8z"/><path class="st8" d="M51 30.6c-2.6-2.6-2.5-6.7 0-9.3 2.6-2.6 6.7-2.5 9.3 0 2.6 2.6 2.5 6.7 0 9.3-2.6 2.6-6.7 2.6-9.3 0z"/><path class="st7" d="M51.4 21.8c1.5-1.5 3.5-2.3 5.4-2.2-2-.4-4.2.2-5.8 1.8-1.6 1.6-2.2 3.8-1.8 5.8 0-1.9.7-3.9 2.2-5.4z"/><path class="st9" d="M60.3 21.4c-.2-.2-.4-.4-.6-.5 2 2.6 1.8 6.3-.6 8.7-2.3 2.3-6.1 2.4-8.6.4.2.2.3.4.5.6 2.6 2.6 6.7 2.6 9.3 0 2.6-2.5 2.6-6.7 0-9.2z"/><path class="st5" d="M18 63.2c-.5-.6-.6-1.4-.1-2l22.5-26.3c1.7-1.9 4.7-2 6.5-.2 1.8 1.8 1.7 4.8-.3 6.5L20 63.2c-.6.5-1.4.5-2 0z"/><path class="st10" d="M47 34.6l-.2-.2L18 63.1c.5.5 1.4.6 2 .1l26.7-22.1c2-1.6 2.1-4.6.3-6.5z"/><g><path class="st2" d="M17.4 63.8c-.9-.9-.9-2.3-.1-3.3l22.5-26.3c1-1.1 2.4-1.8 3.9-1.8s2.9.5 4 1.6c1 1.1 1.6 2.5 1.6 4-.1 1.5-.8 2.9-1.9 3.9l-26.8 22c-.9.8-2.3.8-3.2-.1zm28.9-28.5c-.7-.7-1.6-1.1-2.6-1-1 0-1.9.5-2.5 1.2L18.6 61.7c-.2.2-.2.6 0 .8.2.2.6.2.8 0l26.7-22.1c.8-.6 1.2-1.5 1.2-2.5.1-1-.3-1.9-1-2.6z"/></g><g><path class="st7" d="M56.5 21.4c-.1-.1-.2-.3-.2-.5 0-.3.3-.6.7-.5.5.1.6.1 1 .3l.1.1c.3.1.4.5.3.8-.1.3-.5.4-.8.3l-.1-.1c-.3-.2-.3-.2-.6-.2-.2-.1-.3-.1-.4-.2z"/></g><g><path class="st7" d="M52 22.6c-.2-.3-.2-.7 0-.9.1-.1 1.7-1.5 3.7-1.5.3 0 .6.3.6.6s-.3.6-.6.6c-1.5 0-2.9 1.2-2.9 1.2-.2.2-.5.2-.8 0z"/></g><g class="st11"><path d="M79 8.5C75.2 18.6 68.8 30.2 57.9 41c-8.4 8.4-16.8 12.9-24.8 17 .7.8 1.4 1.4 1.7 1.7 9.2-3.1 17.1-7.3 25.8-16C72.4 32.1 78.1 19.7 79 8.5z"/></g><path class="st7" d="M38.2 22.9C48.6 12.5 53.1 9.4 62.1 5.2 57.3 7 48.5 10.5 37.9 21c-8.7 8.6-12.9 16.5-16 25.7l.2.2c3.9-7.7 8-16 16.1-24z"/></svg>
```

File: web/assets/themes/ezplatform/images/platform-sh-rocket.svg [hunk 2]
```diff
@@ -0,0 +1,294 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SylvainGuittard) [inline web/assets/themes/ezplatform/images/platform-sh-rocket.svg]: Same here 
```
<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 84 65"><style>.st0{fill:#0A3B4C;} .st1{fill:#A4B5BC;} .st2{fill:#F15A22;} .st3{fill:#404040;} .st4{opacity:5.000000e-02;} .st5{fill:#FFFFFF;} .st6{fill:#DBDBDB;} .st7{fill:#545454;} .st8{opacity:0.1;} .st9{opacity:0.3;} .st10{fill:none;}</style><path class="st0" d="M16.7 54c.5-.4 1.1-.6 1.8-.6 1.2 0 2.1.7 2.6 1.6.6-.9 1.7-1.5 2.9-1.5 1.9 0 3.4 1.5 3.6 3.4.3-.1.7-.2 1-.2 2 0 3.6 1.6 3.6 3.6 0 .3 0 .6-.1.8.4-.1.8-.2 1.2-.2 1.8 0 3.2 1.3 3.5 3h.3c.8 0 1.5.2 2.1.7.4-1.5 1.8-2.7 3.5-2.7 1.2 0 2.3.6 2.9 1.5.2-.1.5-.1.7-.2.5-.8 1.4-1.3 2.5-1.3h.2c.5-.8 1.4-1.3 2.5-1.3.3 0 .5 0 .8.1.1-1.5 1.4-2.7 2.9-2.7.4 0 .7.1 1.1.2V58c0-2.1 1.7-3.9 3.9-3.9.8 0 1.6.3 2.2.7.5-.8 1.4-1.4 2.5-1.4.4 0 .9.1 1.3.3.4-1.1 1.5-1.9 2.7-1.9h.4c2.1-4.1 3.3-8.8 3.3-13.8C72.5 21.2 58.8 7.5 42 7.5S11.5 21.2 11.5 38c0 5.5 1.4 10.6 4 15 .5.2.9.6 1.2 1z"/><path class="st1" d="M16.7 54c.5-.4 1.1-.6 1.8-.6 1.2 0 2.1.7 2.6 1.6.6-.9 1.7-1.5 2.9-1.5 1.9 0 3.4 1.5 3.6 3.4.3-.1.7-.2 1-.2 2 0 3.6 1.6 3.6 3.6 0 .3 0 .6-.1.8.4-.1.8-.2 1.2-.2 1.8 0 3.2 1.3 3.5 3h.3c.8 0 1.5.2 2.1.7.4-1.5 1.8-2.7 3.5-2.7 1.2 0 2.3.6 2.9 1.5.2-.1.5-.1.7-.2.5-.8 1.4-1.3 2.5-1.3h.2c.5-.8 1.4-1.3 2.5-1.3.3 0 .5 0 .8.1.1-1.5 1.4-2.7 2.9-2.7.4 0 .7.1 1.1.2V58c0-2.1 1.7-3.9 3.9-3.9.8 0 1.6.3 2.2.7.5-.8 1.4-1.4 2.5-1.4.4 0 .9.1 1.3.3.4-1.1 1.5-1.9 2.7-1.9h.4c.8-1.5 1.4-3.1 1.9-4.7-.7.1-1.4.6-1.7 1.2-.3-.1-.7-.2-1.1-.2-1.4 0-2.6 1-2.9 2.4-.3-.1-.5-.1-.8-.1-.8 0-1.5.3-2 .8-.7-.5-1.5-.8-2.4-.8-2.1 0-3.9 1.7-3.9 3.9v.6c-.4-.2-.8-.3-1.2-.3-1.5 0-2.7 1.1-2.9 2.6-.4-.2-.8-.3-1.2-.3-1 0-1.9.5-2.5 1.3h-.2c-1 0-1.9.5-2.5 1.3-1 .1-1.8.8-2.2 1.6-.4-.2-.9-.4-1.4-.4-1 0-1.9.5-2.4 1.3-.6-1.1-1.8-1.9-3.1-1.9h-.3c-.3-1.7-1.7-3-3.5-3-.4 0-.8.1-1.2.2.1-.3.1-.5.1-.8 0-2-1.6-3.6-3.6-3.6-.4 0-.7.1-1 .2-.1-1.9-1.7-3.4-3.6-3.4-1.2 0-2.2.6-2.9 1.5-.5-1-1.5-1.6-2.6-1.6-.7 0-1.3.2-1.8.6-.4-.7-1.2-1.1-2.1-1.1h-.5c.5 1.3 1.1 2.6 1.8 3.8.4.3.8.7 1.1 1.1z"/><path class="st2" d="M35.1 16h13.5v31.7H35.1z"/><path class="st3" d="M41.8.5c-1.5.9-2.8 2.3-3.9 4.1h7.8c-1-1.8-2.4-3.2-3.9-4.1z"/><path class="st2" d="M48.6 16c0-4.1-1.1-8.6-2.9-11.4h-7.8C36.1 7.5 35 11.9 35 16h13.6z"/><g class="st4"><path class="st3" d="M41.8 16h6.7v31.7h-6.7zM45.7 4.6c-1.1-1.7-2.4-3.1-3.9-4.1V16h6.7c.1-4.2-1-8.6-2.8-11.4z"/></g><path class="st5" d="M29.7 20h5.4v30h-5.4z"/><path class="st2" d="M29.7 48.5h5.4v.6h-5.4z"/><path class="st5" d="M32.4 11.9c-1.6 1.4-2.7 4.5-2.7 8.1h5.4c0-3.6-1.1-6.7-2.7-8.1z"/><path class="st6" d="M29.7 27.5h5.4v1.3h-5.4zM29.7 30.2h5.4v1.3h-5.4zM29.7 20h5.4v1.3h-5.4z"/><path class="st5" d="M48.6 20H54v30h-5.4z"/><path class="st2" d="M48.6 48.5H54v.6h-5.4z"/><path class="st5" d="M51.3 11.9c-1.6 1.4-2.7 4.5-2.7 8.1H54c0-3.6-1.1-6.7-2.7-8.1z"/><path class="st6" d="M48.6 20H54v1.3h-5.4z"/><g><path class="st6" d="M48.6 27.5H54v1.3h-5.4z"/></g><g><path class="st6" d="M48.6 30.2H54v1.3h-5.4z"/></g><g class="st4"><path class="st3" d="M54 20c0-3.6-1.1-6.7-2.7-8.1V50H54V20z"/></g><g class="st4"><path class="st3" d="M35.1 20c0-3.6-1.1-6.7-2.7-8.1V50h2.7V20z"/></g><g><path class="st3" d="M49.3 50.1h4v.3h-4z"/></g><g><path class="st7" d="M53.6 51.1h-4.7l.4-.7h4z"/></g><g><path class="st3" d="M30.4 50.1h4v.3h-4z"/></g><g><path class="st7" d="M34.8 51.1H30l.4-.7h4z"/></g><g><path class="st5" d="M35.4 29.5c0 9.4-9.3 6.7-9.3 17.5h12V18.7s-2.7 1.3-2.7 10.8z"/><path class="st3" d="M31.4 38.1c2.3-1.8 4.7-3.7 4.7-8.6 0-4.8.7-7.4 1.3-8.8v-1.3c-.8 1.1-2 3.8-2 10.1 0 9.4-9.4 6.7-9.4 17.5h.7v-.7c.2-4.7 2.3-6.4 4.7-8.2z"/><path class="st5" d="M48.2 29.5c0 9.4 9.2 6.7 9.2 17.5H45.6V18.7c-.1 0 2.6 1.3 2.6 10.8z"/><path class="st3" d="M52.3 38.1c-2.3-1.8-4.7-3.7-4.7-8.6 0-4.8-.7-7.4-1.3-8.8v-1.3c.8 1.1 2 3.8 2 10.1 0 9.4 9.4 6.7 9.4 17.5H57v-.7c-.2-4.7-2.4-6.4-4.7-8.2z"/><path class="st5" d="M37.1 24.1c0-7.5 2.1-13.5 4.7-13.5s4.7 6 4.7 13.5h-9.4zM37.1 32.2h9.4v16.2h-9.4zM37.1 24.1h9.4v8.1h-9.4z"/><path class="st3" d="M37.1 24h9.4v.2h-9.4zM37.1 32.1h9.4v.2h-9.4zM37.1 40.2h9.4v.2h-9.4z"/><g><path class="st3" d="M43.6 15.8c-1.1-.4-2.4-.4-3.5 0l-.4-.9c1.3-.5 2.9-.5 4.2 0l-.3.9z"/><path class="st5" d="M41.7 18.7l-1.4-5.4h.2l1.4 5.4z"/><path class="st5" d="M41.9 18.7h-.2l1.4-5.4h.2z"/></g><g><path class="st3" d="M52.6 45.6h4.3v.2h-4.3z"/></g><g><path class="st3" d="M26.7 45.6H31v.2h-4.3z"/></g><g><path class="st3" d="M37.8 48.4h2.7v.3h-2.7z"/></g><g><path class="st7" d="M40.8 49.4h-3.3l.3-.7h2.7z"/></g><g><path class="st3" d="M43.2 48.4h2.7v.3h-2.7z"/></g><g><path class="st7" d="M46.2 49.4h-3.4l.4-.7h2.7z"/></g><g class="st8"><path class="st3" d="M41.8 10.6v37.8h4.7V24.1c.1-7.5-2.1-13.5-4.7-13.5z"/></g><g><path class="st7" d="M41.8 41.5c-.2.3-.3 1.4-.3 2v5.6c0 .2.2.3.3.3.2 0 .3-.2.3-.3v-5.6c.1-.6-.1-1.8-.3-2z"/></g></g><g class="st9"><path class="st10" d="M26 47h3.7v-8.5c-2 1.8-3.7 3.9-3.7 8.5zM45.6 16h-7.5 3.7z"/><path class="st3" d="M38.1 16h-.4.4zM45.6 16h.4-.4z"/><path class="st7" d="M41.8 10.6c1.5 0 2.9 2.1 3.8 5.4h.4c-.9-3.6-2.4-5.7-4.1-5.7-1.7 0-3.2 2.2-4.1 5.7h.4c.7-3.3 2.1-5.4 3.6-5.4zM29.7 38.6c2.6-2.3 5.7-3.7 5.7-9.1V29v-.1-.4-.1-.5-.4-.1-.3-.1-.4-.1-.4-.1-.3-.1-.4-.1-.3-.1-.3-.1c0-.1 0-.2.1-.3v-.1-.2-.1-.2-.1c0-.1 0-.2.1-.3v-.1c0-.1 0-.1.1-.2v-.1c0-.1 0-.1.1-.2 0-.1 0-.2.1-.2v-.1c0-.1 0-.1.1-.2v-.1c0-.1 0-.1.1-.2 0-.1 0-.1.1-.2v-.1c0-.1 0-.1.1-.1v-.1c0-.1 0-.1.1-.2 0-.1.1-.1.1-.2 0 0 0-.1.1-.1 0-.1.1-.1.1-.2 0 0 0-.1.1-.1 0 0 0-.1.1-.1l.1-.1s0-.1.1-.1c0 0 0-.1.1-.1.2-1.2.4-2.4.7-3.4h-.4c-.3 1-.5 2.1-.6 3.3 0 0 0 .1-.1.1 0 0 0 .1-.1.1l-.1.1s0 .1-.1.1l-.1.1s-.1.1-.1.2c0 0 0 .1-.1.1v.1c0 .1-.1.1-.1.2s0 .1-.1.2v.1s0 .1-.1.2c0 .1-.1.1-.1.2s0 .1-.1.2v.1c0 .1 0 .1-.1.2 0 .1 0 .2-.1.3 0 .1 0 .1-.1.2v.1c0 .1 0 .1-.1.2 0 .1 0 .2-.1.3v.5c0 .1 0 .2-.1.3V29.3c0 4.4-2.1 6.1-4.4 7.8l-1 1.5zM29.7 47h7.4v.3h-7.4zM46.6 47H54v.3h-7.4z"/><path class="st7" d="M46.3 19.4c0 .1 0 .1 0 0l.1.1s0 .1.1.1l.1.1s0 .1.1.1c0 0 0 .1.1.1 0 0 .1.1.1.2 0 0 0 .1.1.1 0 .1.1.1.1.2s0 .1.1.2v.1s0 .1.1.1v.1c0 .1 0 .1.1.2 0 .1 0 .1.1.2v.1c0 .1 0 .1.1.2v.1c0 .1 0 .2.1.2 0 .1 0 .1.1.2v.1c0 .1 0 .1.1.2v.1c0 .1 0 .2.1.3V23.5c0 .1 0 .2.1.3V29c0 5.4 3.1 6.8 5.7 9.1v-.4c-.3-.3-.7-.6-1-.8-2.2-1.8-4.4-3.4-4.4-7.8v-.7-.5-.5-.6-.5-.4-.5-.4-.4-.3-.1-.4c0-.1 0-.2-.1-.3V23.2 23c0-.1 0-.2-.1-.3 0-.1 0-.1-.1-.2v-.1c0-.1 0-.1-.1-.2 0-.1 0-.2-.1-.3 0-.1 0-.1-.1-.2v-.1c0-.1 0-.1-.1-.2 0-.1-.1-.1-.1-.2s0-.1-.1-.2v-.1c0-.1-.1-.1-.1-.2s-.1-.1-.1-.2v-.1s0-.1-.1-.1c0-.1-.1-.1-.1-.2L47 20s0-.1-.1-.1l-.1-.1s0-.1-.1-.1c0 0 0-.1-.1-.1-.2-1.5-.4-2.6-.6-3.6h-.4c.3 1 .5 2.2.7 3.4z"/></g><path class="st7" d="M30 51.1z"/><path class="st5" d="M33.3 60.9c1.8 0 3.2 1.3 3.5 3h.3l-2.5-12.8H30l-.8 5.7c1.7.3 3 1.7 3 3.5 0 .3 0 .6-.1.8.4-.2.8-.2 1.2-.2zM55 57.9L53.7 51H49l-1.5 11.1c.4-.2.9-.4 1.4-.4h.2c.5-.8 1.4-1.3 2.5-1.3.3 0 .5 0 .8.1-.1-1.3 1.1-2.5 2.6-2.6zM17.2 29.8l.4.8.8.1-.6.6.2.9-.8-.4-.8.4.1-.9-.6-.6.9-.1zM60 17.5l.4.8.9.1-.6.7.1.8-.8-.4-.8.4.2-.8-.6-.7.8-.1zM26 20.1l.2.4.4.1-.3.3.1.4-.4-.2-.4.2.1-.4-.4-.3.5-.1zM64.1 36.8l.2.4h.4l-.3.4.1.4-.4-.2-.4.2v-.4l-.3-.4h.5zM21.5 44.5l.3.6.6.1-.4.5.1.6-.6-.3-.6.3.1-.6-.5-.5.7-.1z"/></svg>
```

Comment (Plopix) [reply web/assets/themes/ezplatform/images/platform-sh-rocket.svg]: done!

**Comment by SylvainGuittard** [comment]
Thank you so much @Plopix

**Pull request merged**